sdf-framework.cddl

```diff
@@ -1,7 +1,7 @@
 start = sdf-syntax
 
 sdf-syntax = {
- info: sdfinfo                  ; don't *require* this in flexible syntax, though
+ ? info: sdfinfo                  ; This will be required in most process policies, but not a syntax error
```

Comment:
although not a syntax error, a warning should be given that this block is missing